API/Services/Implementations/PlayerService.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace API.Services.Implementations;
 
-public class PlayerService(IPlayersRepository playerRepository, IMapper mapper) : IPlayerService
+public class PlayerService(IPlayersRepository playerRepository, IUserService userService, IMapper mapper) : IPlayerService
```

Comment:
Assuming the endpoint is supposed to be on the `UsersController`, the changes in this file would be discarded.